src/antidote_crdt_orset.erl

```diff
@@ -108,33 +108,33 @@ value({fragment, Elem}, ORSet) ->
 value({tokens, Elem}, ORSet) ->
     case orddict:find(Elem, ORSet) of
         error ->
-            orddict:new();
+            [];
```

Comment:
I changed it because I think we are returning a list of tokens and not a dictionary.